zubhub_frontend/zubhub/src/components/input/inputScripts.js

```diff
@@ -4,7 +4,7 @@ export const refactorVideoUrl = url => {
     return url.replace('watch?v=', 'embed/');
   } else {
     if (url.includes('youtu.be')) {
-      return 'https://www.youtube.com/embed/'.concat(url.split('/')[3]);
+      return 'https://www.youtube-nocookie.com/embed/'.concat(url.split('/')[3]);
```

Comment:
you should do the same thing for the first if conditional `if (url.includes('youtube.com')) {`

Reply:
copied!